Makefile

```diff
@@ -126,7 +123,7 @@ test-e2e: e2e-setup
 	$(info Running E2E test: $@)
 	$(Q)GO111MODULE=on operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --up-local --go-test-flags "-v -timeout=15m"
 
-.PHONY: test-unit
+.PHONY: test-e2e
```

Comment:
It's `unit`. Please revert.

Reply:
@otaviof This target is to run unit test. You can see `test-e2e` target just above it.

Reply:
Yes - they are 2 separate tests.